src/storage-types/build.rs

```diff
@@ -80,6 +80,8 @@ use std::env;
 fn main() {
     env::set_var("PROTOC", protobuf_src::protoc());
 
+    const ATTR: &str = "#[derive(Eq, PartialOrd, Ord, ::serde::Serialize, ::serde::Deserialize)]";
```

Comment:
Maybe add `"#[derive(::proptest_derive::Arbitrary)]"` here? Up to you though

Reply:
Done.